docker/Dockerfile.cpu

```diff
@@ -1,31 +1,47 @@
-FROM ubuntu:18.04
+FROM ubuntu:18.04 AS compile-image
 
 ENV PYTHONUNBUFFERED TRUE
 
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
-    fakeroot \
     ca-certificates \
-    dpkg-dev \
     g++ \
     python3-dev \
-    openjdk-11-jdk \
+    python3-distutils \
+    python3-venv \
+    openjdk-11-jre-headless \
     curl \
-    vim \
     && rm -rf /var/lib/apt/lists/* \
     && cd /tmp \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python3 get-pip.py
 
+RUN python3 -m venv /home/venv
+
+ENV PATH="/home/venv/bin:$PATH"
+
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 RUN update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
 
-RUN pip install --no-cache-dir psutil
-RUN pip install --no-cache-dir torch
-RUN pip install --no-cache-dir torchvision
-RUN pip install --no-cache-dir torchtext
-ADD serve serve
-RUN pip install ../serve/
+RUN pip install --no-cache-dir torch torchvision torchtext torchserve
+
+FROM ubuntu:18.04 AS runtime-image
```

Comment:
```suggestion
FROM ${BASE_IMAGE AS runtime-image
```

Reply:
Done

Reply:
@seemethere Can you confirm if this meets your expectations for requested changes and then resolve this conversation?